Dockerfile

```diff
@@ -1,19 +1,19 @@
 FROM ruby:2.6.3-alpine
 
+ENV APP_HOME /app
+
 RUN apk add --update build-base postgresql-dev git tzdata nodejs yarn && \
     cp /usr/share/zoneinfo/Europe/London /etc/localtime && \
     echo "Europe/London" > /etc/timezone
 
-RUN gem install bundler:2.0.2
-
-ENV APP_HOME /app
-RUN mkdir $APP_HOME
+COPY . $APP_HOME/
 WORKDIR $APP_HOME
+COPY Gemfile Gemfile.lock package.json yarn.lock $APP_HOME/
 
-ADD Gemfile Gemfile.lock package.json yarn.lock $APP_HOME/
-RUN yarn install --check-files
-RUN bundle install
+RUN gem install bundler:2.0.2 && \
+    yarn install --check-files && \
+    bundle install
 
-ADD . $APP_HOME/
+RUN bundle exec rake assets:precompile
 
-CMD bundle exec rails db:migrate && bundle exec rails server -b 0.0.0.0
```

Comment:
Why are we no longer migrating the database? Will that have to be done manually?

Reply:
+1

Reply:
Addressed in commit 86d4973